app.js

```diff
@@ -1,17 +1,23 @@
 const navAppear = () => {
   const burgerMenu = document.querySelector(".burgerMenu");
+
+  burgerMenu.addEventListener("click", () => {
+    toggleBurgerMenu();
```

Comment:
we could be passing the function directly as opposed to wrap it in an arrow function